packages/flutter_tools/lib/src/fuchsia/fuchsia_ffx.dart

```diff
@@ -93,9 +93,6 @@ class FuchsiaFfx {
       _logger.printError('ffx failed: ${result.stderr}');
       return null;
     }
-    if (result.stderr.contains('No devices found')) {
```

Comment:
How should the test be updated?

https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/general.shard/fuchsia/fuchsia_ffx_test.dart#L45

Reply:
That test shouldn't be updated as `ffx target list --format s` will still give an exit code of 0 when no devices are found. I have added tests for ffx resolving to a specific target.

If we run target list $device_name and $device_name doesn't exist, ffx will return exit code 2.